doc/NotaryContentTrust.md

```diff
@@ -1,26 +1,41 @@
-# Notary Content Trust in IoT Edge
+# **Notary Content Trust for Manifest Trust in IoT Edge**
 **Note: This feature is currently in experimental phase. Its is not released yet and not to be used in production**
 
 Content Trust is achieved in IoT edge with [Notary](https://github.com/theupdateframework/notary) which is based on The Update Framework (TUF). Notary aims to make the internet more secure by making it easy for people to publish and verify content. The goal of Content Trust in IoT edge is to secure the docker container images of the Edge modules with its trust pinned to the Edge device via a root Certificate Authority(CA). When Content Trust is enabled, IoT Edge device only deploys signed images and rejects any other images that are tampered or modified in the transfer over the internet. Feature is supported in Linux OS only for now. 
 
-## Sign and Publish Container Images in Azure Container Registry
-The container images can be signed and published in Azure Container Registry which has an in built Notary support for content trust. 
-
+### To try the feature, follow the 4 steps below
+1. **Set up the necesarry certificates**
+    * Install the tools
+    * Generate certificates
+        * Create the root CA thats pins to the edge device 
+        * Create the root certificates for each container images that chains to the root CA
+2. **Initialize TUF Trust Collection using Notary client**
+    * Notary Installation
+    * Notary Configuration
+    * Notary TUF Trust collection Intialization
+    * Notary Snapshot key rotation
+3. **Sign and publish images using Docker client**
+    * Docker Environment variable configuration
+    * Sign and publish 
+4. **Enable Content Trust in IoT edge device**
+    
+## Step 1. Set up the necessary certificates 
+
+##### a. Install the tools
 To publish signed images, following tools must be installed in a publisher's environment which is different from the Edge device. 
 1. OpenSSL - to generate root CA for each registry and root certs for each container
 2. Notary client - to initialize and create TUF trust collection
 3. Docker client - to sign the image and publish to the Container Registry
 
-
-### 1. Generate Certificates using OpenSSL
+##### b. Generate Certificates 
 Follow instructions in [iotedge repository]([https://github.com/ggjjj/iotedge/blob/master/edgelet/doc/devguide.md](https://github.com/ggjjj/iotedge/blob/master/edgelet/doc/devguide.md)) to install OpenSSL. There are two types of certificates to be generated.
 1. root CA for each container registry
 2. root certificate for each container image
 
 There is only one root CA needed for each container registry which forms the root of all trust for all its container images. When using multiple registries, multiple root CAs have to be generated. Root certificate for each image is needed to initialize the TUF trust collection using Notary.
```

Comment:
Add a note or caveat to the statement to indicate root CA per registry is a scoping choice in this experimental release.  That a full-fledged feature would consider consolidating per registry roots into a unified root.